.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'space-before-function-paren': 0,
-    'no-unused-vars': ['warn', { varsIgnorePattern: '^_' }],
+    'no-unused-vars': 'off',
```

Comment:
recommendation is to disable `no-unused-vars` as it complains about unused vars in interface declarations